README.md

```diff
@@ -87,4 +87,7 @@ The most common mood is :-)
 
 **think**: Should you use an array or a hash to solve this problem? Why?
 
+I believe that arrays are more useful when using data and it involves collecting a collection of elements.
```

Comment:
This can be a true statement. And arrays *could* have been used, but I would need a good reason why. I expected you all to pick hash, which you and I discussed in person :)